### ISSUE TITLE
Migrate any documentation from atomic wiki into the core documentation

### ISSUE THREAD

**Issue by joewiz** [eXist-db/documentation#102]
From @dizzzz on eXist-open, referring to http://atomic.exist-db.org/HowTo/JettySSL:

> that document is from 2008 ! it is probably outdated :-/ (classic issue with wikis)

**Comment by joewiz** [comment]
Does anyone have a list of the available atomic wiki articles that we could consider as candidates for inclusion in the documentation? (Unfortunately [the list of articles in the "HowTo" section](http://atomic.exist-db.org/exist/apps/wiki/HowTo/) does not link to the articles.)  :(

**Comment by duncdrum** [comment]
@joewiz is this still an issue?

**Comment by joewiz** [comment]
@duncdrum Yes, why do you ask? I was proposing that we consider articles such as http://atomic.exist-db.org/HowTo/JettySSL as candidates for inclusion in the eXist core documentation.  Has that been folded in? 

Or did you mean the atomicwiki link issue I mentioned in my 2nd post? I'm still unable to load the list of all "HowTo" articles. Loading http://atomic.exist-db.org/HowTo returns a message, "Either the feed is empty or you are not allowed to view the entries".

**Comment by dizzzz** [comment]
That SSL article is not relevant anymore as exist 3+ has the HTTPS port open by default,,,, mentioning that is it on port 8443 should be sufficient?

**Comment by duncdrum** [comment]
Ok then, here is a list of documentation candidates from the wiki:
- How-To: 
  - [ ] OxygenXml
  - [ ] SQLDatabases
  - [ ] Troubleshooting
  - [ ] Wrapper
  - [ ] XQTS
  - [ ] XSLT2
  - [ ] ~~JettySSL~~
-  Blogs: 
   - [ ]  AkismetAPI
   - [ ]  AsirraAPi
   - [ ]  BaiscAuthApps
   - [ ]  Captcha
   - [ ]  ConsistencyChecker
   - [ ]  ContentExtraction
   - [ ]  EclipsePlugin
   - [ ]  FLOSS
   - [ ]  H(igher)o(rder)F(unction)
   - [ ]  InMemoryFragments
   - [ ]  IntroducingXqueryRewrite
   - [ ]  JSON Serializer
   - [ ]  KWICSearch
   - [ ]  LuceneIndexModule
   - [ ]  NewIndexing
   - [ ]  NewRangeIndex
   - [ ]  NewXQuery3
   - [ ]  RESTfulServices
   - [ ]  TipsnTricks
   - [ ]  WarningBadMemory
   - [ ]  XQTS994
   - [ ]  XQuery3.1
   - [ ]  XQueryDashboard
   - [ ]  XQueryMap
   - [ ]  TravisCI
   - [ ]  FormValidation

**Comment by duncdrum** [comment]
@adamretter SQL, and the triplet of captcha akisnet, asirra articles are from you any thoughts if they should go into the docs?

@joewiz a number of articles have been moved some time ago, but I can find no trace of the decision process, so its possible that the ones that weren't, didn't make it into the docs for a reason. 

@dizzzz some of this stuff seems to be your kind of thing (eclipse, FLOSS?) any idea if and what should get moved

The only immediate candidate to me is the article on configuring TravisCI.

**Comment by eriksiegel** [comment]
@duncdrum I'll await the results of this discussion. When i have time left after doing what is now the core documentation I could pick up some of the articles for editing and inclusion.

**Comment by duncdrum** [comment]
Update for better legibility:
@adamretter your wiki articles that need a vote:
  -  SQLDatabases
   -  AkismetAPI
   -  AsirraAPi
   -  Captcha
   -   Puppet

@dizzzz:
   -   TipsnTricks

@JoernT:
   -   FormValidation

other:
   -  EclipsePlugin
   -  TravisCI

**Comment by duncdrum** [comment]
see eXist-db/exist#1778

**Comment by adamretter** [comment]
@duncdrum these should stay on the Wiki as they are more example code:
1. AkismetAPI
2. AsirraAPi
3. Captcha

Where can I find the Puppet and SQL Databases articles to check?

**Comment by duncdrum** [comment]
@adamretter [https://exist-db.org/exist/apps/wiki/HowTo/SQLDatabases/](https://exist-db.org/exist/apps/wiki/HowTo/SQLDatabases/)

**Comment by duncdrum** [comment]
Outstanding wiki articles: 5/33
  -   SQLDatabases 
  -   Puppet
  -   TipsnTricks
  -   EclipsePlugin
  -   TravisCI

**Comment by joewiz** [comment]
@duncdrum By "outstanding" (in https://github.com/eXist-db/documentation/issues/102#issuecomment-389493850), these are the 5 remaining wiki articles that need to be adapted to the documentation, correct?

**Comment by duncdrum** [comment]
Yup

**Comment by line-o** [comment]
The connection to SQLDatabases was recently updated to allow pooling.
Documenting that would be a great opportunity to get one step closer to closing this issue.

**Comment by adamretter** [comment]
@line-o Already documented - https://github.com/eXist-db/documentation/pull/608

**Comment by line-o** [comment]
Now that you said it :) I remember reviewing that.
So ... is this issue still valid?
